### ISSUE TITLE
A missing token (typo)

### ISSUE THREAD

**Issue by Hilla-Merhav** [IAHLT/UD_Hebrew#18]
@amir-zeldes
We know how to report a redundant token (`reparandum`). Do we have a way to report a missing one?
I have "במטרה להעלות את נפח דם" instead of "את נפח **ה**דם". We make sure to report typos when they happen inside a token, do we have any way to report then when they omits tokens?

**Comment by amir-zeldes** [comment]
No, there is currently no way to indicate a word that is just left out, see other people wanting to do this here:

https://github.com/UniversalDependencies/docs/issues/801

This remains to be standardized sadly.

**Comment by Hilla-Merhav** [comment]
@amir-zeldes 
Do we mark this kind of sentences 'corrupt'? 
I am not sure, because on one hand I assume we don't want to lose a sentence just because of a missing token, but on the other hand, we make sure to indicate "הצלחנ" is not another legitimate form of the lemma "הצלחה", so I assume we don't want to educate the parser that "את" is legitimate case for an indefinite NOUN.

**Comment by amir-zeldes** [comment]
I don't think of the sentence as corrupt, in both speech and writing we have a lot of structures that are not considered 'standard', but if they recur often enough (and systematically) then maybe they are just alternative constructions. We have some suggestions on how to mark up non-canonical language as well as an overview of what other UD treebanks are doing here:

https://arxiv.org/pdf/2011.02063.pdf

But omitted tokens are still a blind spot, hence the docs issue above. The closest *existing* practice would be to place `CorrectForm=הדם` on the token דם, but of course that doesn't accurately convey that we are missing a token. In GUM such cases are annotated outside conllu using a TEI <sic> tag in XML, and we intend to include that in MISC in UD release 2.9 like this:

`XML=<sic></sic>`

But that's also not a full solution.
